tests/cases/conformance/types/spread/spreadOverwritesPropertyStrict.ts

```diff
@@ -0,0 +1,15 @@
+// @strict: true
+declare var ab: { a: number, b: number };
+declare var abq: { a: number, b?: number };
+var unused1 = { b: 1, ...ab } // error
+var unused2 = { ...ab, ...ab } // ok, overwritten error doesn't apply to spreads
+var unused3 = { b: 1, ...abq } // ok, abq might have b: undefined
```

Comment:
It would be nice to add

```ts
var unused4 = { ...ab, b: 1 };
var unused5 = { ...abq, b: 1 };
```

just so the scope of the new error is clear to someone reading this test (me) (these should not be errors, right?)

Reply:
Yep, those are not errors. I added the cases.